scala/ppml/src/main/java/com/intel/analytics/bigdl/ppml/FLHelper.java

```diff
@@ -24,6 +24,8 @@
     // Client property
     String clientTarget = "localhost:8980";
     String taskID = "taskID";
+    String certChainFilePath = "";
```

Comment:
Add default value, such as `server.pem` and `server.crt`

Reply:
It should be null for tls normal.